.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Install and Build
```

Comment:
Do we want / need to cache? https://github.com/actions/cache/blob/main/examples.md#node---npm

Reply:
We probably don't need to, but I can go ahead and add it. Currently, our workflow runs in just under half a second.